lib/pages/chat/chat.dart

```diff
@@ -114,6 +114,8 @@ class ChatController extends State<Chat>
 
   final getPinnedMessageInteractor = getIt.get<ChatGetPinnedEventsInteractor>();
 
+  StreamSubscription? getPinnedMessageStreamSubscription;
```

Comment:
what is this change?

Reply:
I just forgot to remove it, but it was to be able to cancel getPinnedMessageInteractor's stream subscription. I finally found out that getPinnedMessageInteractor is not used at all.